src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/RandomData.java

```diff
@@ -0,0 +1,655 @@
+package com.microsoft.sqlserver.jdbc.AlwaysEncrypted;
+import java.math.BigDecimal;
```

Comment:
yep. randomData and Util were originally for AE only, but I think they can be used elsewhere as well - i moved both of them to util package.